### PR TITLE
login screen layout

### PR DESCRIPTION
**Pull request by fraagu10**
Includes a navigation bar with basic links to help you navigate through the prototype of the app.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,9 +19,12 @@
     "@angular/compiler": "~12.2.0",
     "@angular/core": "~12.2.0",
     "@angular/forms": "~12.2.0",
+    "@angular/localize": "~12.2.0",
```
Comment (pieroapretto) [inline package.json:22]: What's the strategy behind adding this package?

File: src/app/app.component.html [hunk 2]
```diff
@@ -1,501 +1,25 @@
...
-
-<!-- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * -->
-<!-- * * * * * * * * * * * The content above * * * * * * * * * * * -->
-<!-- * * * * * * * * * * is only a placeholder * * * * * * * * * * -->
-<!-- * * * * * * * * * * and can be replaced. * * * * * * * * * * * -->
-<!-- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * -->
-<!-- * * * * * * * * * * End of Placeholder * * * * * * * * * * * -->
-<!-- * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * -->
+<nav class="navbar navbar-expand-lg navbar-light">
+    <!-- <a class="navbar-brand" href="#">Navbar</a> -->
```
Comment (pieroapretto) [inline src/app/app.component.html]: Are these comments needed?

File: src/app/components/login/login.component.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      <div class="row d-flex justify-content-center align-items-center h-100">
+        <div class="col-12 col-md-8 col-lg-6 col-xl-5">
+          <div class="card shadow-2-strong" style="border-radius: 1rem;">
+            <div class="card-body p-5 text-center">
+  
+              <h3 class="mb-5">Sign in</h3>
+  
+              <div class="form-outline mb-4">
+                <input type="email" id="typeEmailX-2" class="form-control form-control-lg" placeholder="Email"/>
+                <!-- <label class="form-label" for="typeEmailX-2">Email</label> -->
```
Comment (pieroapretto) [inline src/app/components/login/login.component.html]: Is this comment needed?

File: src/app/components/login/login.component.html [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+              </div>
+
+              <div class="form-outline mb-4">
+                <input type="password" id="typePasswordX-2" class="form-control form-control-lg" placeholder="Password" />
+                <!-- <label class="form-label" for="typePasswordX-2">Password</label> -->
```
Comment (pieroapretto) [inline src/app/components/login/login.component.html]: Is this comment needed?

File: src/app/components/pantry/pantry.component.html [hunk 5]
```diff
@@ -0,0 +1 @@
+<p>pantry works!</p>
```
Comment (pieroapretto) [inline src/app/components/pantry/pantry.component.html:1]: 💯

**Review by pieroapretto — COMMENTED** [review]
Nice work so far! 🥇

**Comment by pieroapretto** [comment]
@fraagu10 can you address the comments I made on this PR?

File: package.json (hunk 1 above)
Comment (ghost) [reply package.json:22]: Angular documentation says that it is used to prepare the project for usage in different locales (formatting dates, currencies, and time). I did not personally implement this so I will be leaving it in for the mean time.

**Review by pieroapretto — APPROVED** [review]
Looks good to me 👍

**Comment by ghost** [comment]
Am I set to merge this branch?

**Comment by pieroapretto** [comment]
> Am I set to merge this branch?

yes

**Pull request merged**